theme/templates/email/email_verify_subject.txt

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+{% trans "Your email in HydroShare has changed" %}
```

Comment:
@mauriel133 Should we change the subject of email to "Update your email address in your account in HydroShare" since the email will not be changed until they confirm? Again I am bringing in @dtarb to finalize the email subject for email update.